src/Microsoft.SqlTools.ServiceLayer/QueryExecution/QueryExecutionService.cs

```diff
@@ -422,6 +392,28 @@ public void InitializeService(ServiceHost serviceHost)
             }
         }
 
+        /// <summary>
+        /// Retrieves the requested subset of rows from the requested result set. Intended to
+        /// </summary>
+        /// <param name="subsetParams">Parameters for the subset to retrieve</param>
+        /// <returns>The requested subset</returns>
+        /// <exception cref="ArgumentOutOfRangeException">The requested query does not exist</exception>
+        public async Task<ResultSetSubset> InterServiceResultSubset(SubsetParams subsetParams)
```

Comment:
>InterService [](start = 43, length = 12)

I'm not quite clear what "InterService" means in this context.  Perhaps a different name (or completed comments :) ) may make it more intuitive.